src/resizable-card-layout.scss

```diff
@@ -0,0 +1,137 @@
+@import "./new-settings";
+@import "./mixins";
+@import "./icon";
+
+$block: #{$fd-namespace}-resizable-card-layout;
+
+.#{$block} {
+  $fd-resizable-card-min-width: 20rem;
+  $fd-resize-handle-width: 0.0625rem;
+  $fd-border-on-drag: 0.125rem dashed var(--sapContent_DragAndDropActiveColor);
+
+  $fd-resizable-card-padding-x: (
+    s: 0.5rem,
+    m: 1rem,
+    l: 3rem,
+  );
+
+  @include fd-reset();
+
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+
+  &__wrapper {
+    @include fd-reset();
+    @include fd-flex();
+
+    min-width: $fd-resizable-card-min-width;
+    margin: 0.5rem;
+    position: relative;
+
+    &__card {
+      @include fd-reset();
+
+      order: 1;
+      flex: 1;
+      position: relative;
+      // subtract width taken by vertical handler and horizontal handler
+      height: calc(100% - #{$fd-resize-handle-width});
+
+      &__icon-wrapper {
+        @include fd-reset();
+
+        width: 1rem;
+        height: 1rem;
+        position: absolute;
+        bottom: 0;
+        right: 0;
+        z-index: 2;
+        padding: 0 0.0625rem 0.0625rem 0;
```

Comment:
can't `0.0625rem` value be reused through variable?

Reply:
fixed. using variable now.